<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SendMessagesEntry</class>
 <widget class="QFrame" name="SendMessagesEntry">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>729</width>
    <height>236</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Sunken</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="spacing">
    <number>12</number>
   </property>
   <item row="3" column="2">
    <layout class="QHBoxLayout" name="sendToLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QValidatedLineEdit" name="sendTo">
       <property name="toolTip">
        <string>The address to send the payment to  (e.g. CKPnZKDzaDXqEgKJ4GdUg58gXxgzA7mkny)</string>
       </property>
       <property name="maxLength">
        <number>34</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="addressBookButton">
       <property name="toolTip">
        <string>Choose address from address book</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../bitcoin.qrc">
         <normaloff>:/icons/address-book</normaloff>:/icons/address-book</iconset>
       </property>
       <property name="shortcut">
        <string>Alt+A</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="pasteButton">
       <property name="toolTip">
        <string>Paste address from clipboard</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../bitcoin.qrc">
         <normaloff>:/icons/editpaste</normaloff>:/icons/editpaste</iconset>
       </property>
       <property name="shortcut">
        <string>Alt+P</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="deleteButton">
       <property name="toolTip">
        <string>Remove this recipient</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../bitcoin.qrc">
         <normaloff>:/icons/remove</normaloff>:/icons/remove</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="messageLabel">
     <property name="text">
      <string>&amp;Message:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
     </property>
     <property name="buddy">
      <cstring>messageText</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Send &amp;To:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>sendTo</cstring>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QValidatedTextEdit" name="messageText">
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="focusPolicy">
      <enum>Qt::WheelFocus</enum>
     </property>
     <property name="tabChangesFocus">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QValidatedLineEdit" name="addAsLabel">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string>Enter a label for this address to add it to your address book</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>&amp;Label:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>addAsLabel</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="publicKeyLabel">
     <property name="text">
      <string>&amp;Public Key:</string>
     </property>
     <property name="buddy">
      <cstring>publicKey</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QValidatedLineEdit" name="publicKey"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QValidatedLineEdit</class>
   <extends>QLineEdit</extends>
   <header>qvalidatedlineedit.h</header>
  </customwidget>
  <customwidget>
   <class>QValidatedTextEdit</class>
   <extends>QPlainTextEdit</extends>
   <header>qvalidatedtextedit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../bitcoin.qrc"/>
 </resources>
 <connections/>
</ui>
